Main.java

```diff
@@ -3,18 +3,18 @@
 public class Main {
 
     public static void main(String[] args) {
-        InvestmentCompany company1 = new InvestmentCompany();
-        InvestmentCompany company2 = new InvestmentCompany("London", 100, 300000, "UK");
-        InvestmentCompany company3 = new InvestmentCompany("Miami",  200, 1000000, "USA",
-                "Accel-KKR", 55, "DAMA", "Hight", "Accel-KKR@gmail.org", 338545455);
+        InvestmentCompany alpineInvestors = new InvestmentCompany();
+        InvestmentCompany frontierCapital = new InvestmentCompany("London", 100, "USA", "UK");
+        InvestmentCompany genesisPark = new InvestmentCompany("Miami",  200, "USA", "USA",
+                "Genesis Park", 55, "DAMA", "Hight", "genesispark@gmail.org", 338545455);
 
-        System.out.println(company1.toString());
-        System.out.println(company2.toString());
-        System.out.println(company3.toString());
+        System.out.println(alpineInvestors.toString());
+        System.out.println(frontierCapital.toString());
+        System.out.println(genesisPark.toString());
         System.out.println("---------------------------------------------");
 
-        InvestmentCompany.setYearOfFoundation(1960);
-        company1.printStaticYearOfFoundation();
-        company1.printYearOfFoundation();
+
+        alpineInvestors.printStaticYearOfFoundation();
```

Comment:
static method should not be referenced from object, it should be referenced from class